src/fibonacciBases.js

```diff
@@ -0,0 +1,48 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibSeq = []
+
+//calculate fibSeq value
+  function fibValue(resultArr){
+    let value = 0
+    for(let i = 0; i < fibSeq.length; i++) {
+      value += ( resultArr[i] * fibSeq[i] )
+    } return value
+  }
+
+//increment fib array
+  function incrementFibLength() {
+    fibSeq.length > 1
+      ? fibSeq.push(fibSeq[fibSeq.length - 1] + fibSeq[fibSeq.length - 2])
+      : fibSeq.push(1)
+  }
+
+//determine fib length
+  function fibLength(number) {
+    while(fibValue(result) < number) {
+      incrementFibLength()
+      result.push(1)
+    }
+  }
+
+//Finalizing results
+  function final() {
```

Comment:
I don't understand this function.  I *think* you're taking the conversion we did yesterday too literally :smile:.

If I were converting a binary number to decimal, I would iterate over each of the digits in the binary number, multiply that digit by the _value of that position_ (e.g. the position at index 3, from the right, of a binary number would represent the value 2^3, or 8), and aggregate that into a total.

The same should be true for conversion from a fibonacci base: iterate over each of the digits of the number, multiply that digit by the _value of that position_ (e.g. the position at index 4 would be fibonacci( 4 ), or 3), and aggregate.